flexget/plugins/list/sonarr_list.py

```diff
@@ -29,7 +29,12 @@ class SonarrSet(MutableSet):
             'include_data': {'type': 'boolean', 'default': False},
             'search_missing_episodes': {'type': 'boolean', 'default': True},
             'ignore_episodes_without_files': {'type': 'boolean', 'default': False},
-            'ignore_episodes_with_files': {'type': 'boolean', 'default': False}
+            'ignore_episodes_with_files': {'type': 'boolean', 'default': False},
+            'profile_id': {'type': 'number', 'default': 1},
+            'season_folder': {'type': 'boolean', 'default': False},
+            'monitored': {'type': 'boolean', 'default': True},
+            'root_folder_path': {'type': 'number', 'default': 1},
```

Comment:
I think this should be an `integer`

Reply:
it doesn't seem to matter between it being a number or an integer typing as both ways work just fine, but I went ahead and switched it and profile_id to be an integer type instead of number type (as it's an int as well going by https://github.com/Sonarr/Sonarr/wiki/Series#post).